server/controllers/entryController.js

```diff
@@ -15,64 +28,82 @@ class entryController {
     Entry.push(data);
-    return res.status(201).json({
-      status: 201,
-      message: 'Entry successfully created',
-      data: {
-        id: data.id,
-        title: data.title,
-        description: data.description,
-        createdOn: data.createdOn,
-      },
-    });
+    data = {
+      id: data.id, title: data.title, description: data.description, createdOn: data.createdOn,
+    };
+    return successResponse(res, 201, 'Entry successfully created', data);
   }
 
+  /**
+   *@description Returns all user's entries
+   * @param {object} req
+   * @param {object} res
+   * @param {number} status
+   * @returns {Array} response
+   */
   static allEntries(req, res) {
     const myentries = Entry.filter(entry => entry.userId === parseInt(req.user.id));
-    return res.status(200).json({ status: 200, data: myentries });
+    return successResponse(res, 200, 'All entries', myentries);
   }
 
+  /**
+   *@description Returns a diary entry of a user
+   * @param {object} req
+   * @param {object} res
+   * @param {number} status
+   * @returns {object} response
+   */
   static getEntry(req, res) {
     const id = parseInt(req.params.entryId);
     const entry = helperFunction.findById(id);
-    return res.status(200).json({ status: 200, data: entry });
+    return successResponse(res, 200, 'One entry', entry);
+    // return res.status(200).json({ status: 200, data: entry });
   }
 
+  /**
+   *@description Modifies a diary entry of a user
+   * @param {object} req
+   * @param {object} res
+   * @param {number} status
+   * @param {string} error
+   * @param {string} message
+   * @returns {object} response
+   */
   static modifyEntry(req, res) {
     const id = parseInt(req.params.entryId);
     const entryIndex = helperFunction.findEntryIndex(id);
     const validateEntry = entrySchema.validate({
       id: Entry[entryIndex].id, title: req.body.title, description: req.body.description,
     });
     if (validateEntry.error) {
-      return res.status(400).json({ status: 400, error: validateEntry.error.details[0].message });
+      return failureResponse(res, 400, validateEntry.error.details[0].message);
     }
     Entry[entryIndex].title = req.body.title;
     Entry[entryIndex].description = req.body.description;
-    return res.status(200).json({
-      status: 200,
-      message: 'Entry successfully edited!',
-      data: {
-        id: Entry[entryIndex].id,
-        title: Entry[entryIndex].title,
-        description: Entry[entryIndex].description,
-        createdOn: Entry[entryIndex].createdOn,
-      },
-    });
+    const entry = {
+      id: Entry[entryIndex].id, title: Entry[entryIndex].title, description: Entry[entryIndex].description, createdOn: Entry[entryIndex].createdOn,
+    };
+    return successResponse(res, 200, 'Entry successfully edited!', entry);
   }
 
+  /**
+   *@description Deletes a diary entry of a user
+   * @param {object} req
+   * @param {object} res
+   * @param {number} status

```

Comment:
`@param` is the parameter for the function defined. for the function `deleteEntry`, you have just two params `req` and `res`. remove the rest

Reply:
status, error and message are not params for this method